plugin/trino-resource-group-managers/src/main/java/io/trino/plugin/resourcegroups/db/DbResourceGroupConfig.java

```diff
@@ -28,6 +29,7 @@
     private String password;
     private boolean exactMatchSelectorEnabled;
     private Duration maxRefreshInterval = new Duration(1, HOURS);
+    private Duration refreshInterval = new Duration(1, SECONDS);
```

Comment:
Let's add an assertion to verify `maxRefreshInterval` > `refreshInterval`. You can refer to `MySqlJdbcConfig` as the example.

Reply:
Is this as you expected? 

I added `public boolean isRefreshIntervalValid()` with AssertTrue,
- https://github.com/trinodb/trino/pull/15389/commits/b89d00bfd261411525d66bd77f615afb2486524c#diff-ac69d75a36cdac0f09c34675ef556da38b8ad8ce4e724bf1764a7645ef252c50R115-R119

Then called `assertTrue(expected.isRefreshIntervalValid());` at TestDbResourceGroupConfig.java#testExplicitPropertyMappings function
- https://github.com/trinodb/trino/pull/15389/commits/b89d00bfd261411525d66bd77f615afb2486524c#diff-d60ec5f89ce45ac2adee416d7287050deacda6122a5db8ccd6252f343944d93cR64

Reply:
Thanks, that's the expected assertion. Can you also add a validation test likes `io.trino.plugin.mongodb.TestMongoClientConfig#testValidation`?

Reply:
Ok added with default setting 
- https://github.com/trinodb/trino/pull/15389/files#diff-d60ec5f89ce45ac2adee416d7287050deacda6122a5db8ccd6252f343944d93cR67-R78

Reply:
The test should verify failure. See `io.trino.plugin.mongodb.TestMongoClientConfig#assertFailsTlsValidation` for more details.

Reply:
Now I understood what you meant :)
https://github.com/trinodb/trino/pull/15389/commits/d371efcc82c5f449fe037016e33312d8223ba2f6